Kor Efret jobs and missions (#5219)
endless-sky/endless-sky: data/korath/korath missions.txt

			`	As you observe your surroundings, a Kor Efret approaches you followed by two others, one likely a child, based on its size, and the other hunched over a cane it uses for support - probably elderly. The one in front tries the Korath language and quickly realizes you don't understand it at all. The Kor Efret pauses to take a deep breath, then erupts into a coughing fit as they inhale far too much smog. It's almost a full minute before they recover.`
			`	The Korath points up to the sky and says, "<planet>, <planet>," while gesturing to the three Korath. They press 500 credits into your hands and continue to mutter in the Korath language. <planet> is the name of the Quarg ringworld in Korath space. They must want transportation to it. 500 credits is shockingly low for any transport job, but based on their look they can't afford to pay you any more. Do you want to take them to <planet>?`
			choice
				`	(Yes.)`

> 				`	(Yes, and give their credits back to them.)`

Reviewer: Ref: #5157 
Need to keep track of this one too.

event "label korath space"
	galaxy "label korath"
		sprite "label/korath"

mission "Translator Development 1A"
	name "Translator Development"
	description "Go to <destination> to ask Eruk if his translation device could be modified for the Wanderers and Korath to use."
	minor
	source "Setar Fort"
	destination "Greenwater"
	to offer
		has "Wanderers: Sestor: Final: done"
	on offer

> 		log "People" "Syciil" `Syciil is a Kor Efret ambassador on Setar Fort. She ensures Ektasi Rek has accomodations and whatever else she may need. Along with Rek, she is investigating ways to improve relations between the Wanderers and Kor Efreti.`

Reviewer 2: sounds less human yea

		conversation
			`When you leave your ship, Rek and a Korath are waiting at the dock. "Hello, Captain <last>. When I was informed you had come to Setar Fort, I came here to see you. This is Scyiil," she says, gesturing to the Korath next to her. Syciil holds up her hands with her palms toward you. "She is an ambassador for the Kor Efreti, and she accompanies me virtually everywhere I go on the planet. I've learned much about the Kor Efreti - and the Korath in general - from her."`
			choice
				`	"Pleased to meet you, Syciil. I'm <first> <last>."`
				`	(Let Rek continue speaking.)`
					goto nogreeting
			`	Rek translates the greeting, and Syciil replies. "She says, 'I am pleased to meet you as well. Rek has told me you've seen much of the galaxy.'`
			label nogreeting

> 			`	"You're probably wondering why I was waiting for you. Syciil and I have been discussing how we might improve our peoples' relations. Obviously, our main hindrance is the language barrier. When I was telling her about our recent effort to shut down the Kor Sestor, she asked how you communicate with other Wanderers who did not speak your language. It's strange that this idea never ocurred to me sooner, but I suppose I've been so busy that I haven't had time to think about it. Could your translation device be converted for Wanderer and Korath languages?"`

Reviewer 3: ```suggestion
			`	Rek continues to speak. "Syciil and I have been discussing how we might improve our peoples' relations. Obviously, our main hindrance is the language barrier. When I was telling her about our recent effort to shut down the Kor Sestor, she asked how you communicate with other Wanderers who did not speak your language. It's strange that this idea never occurred to me sooner, but I suppose I've been so busy that I haven't had time to think about it. Could your translation device be converted for Wanderer and Korath languages?"`
```

			choice

> 				`	"To be honest, I'm not sure. A Hai named Eruk gave me it; he might have an answer. That would be a good place to start."`

Reviewer 3: ```suggestion
				`	"A Hai named Eruk gave the translator to me; he might have an answer."`
```

> 				`	"Sorry, I don't know, and I'm not able to help right now. I'll return when I'm available. Good luck."`
> 					decline

Reviewer 3: There's no reason why the player would want to decline this mission, considering that it has no passenger or cargo or anything requirements.
```suggestion
```

> 			`	"A place to start is all we need. Let me tell Syciil," Rek says, then turns to her and repeats your answer. When she's done, Syciil holds her hands up with her palms toward you and moves them in a circular motion while speaking. "That's a gesture of gratitude. She says, 'Thank you. Your help will benefit both the Wanderers and Kor Efreti.' We would go with you, but both of us are busy with another project right now. Please return to Setar Fort if you make any progress." Rek and Scyiil depart. It sounds like you should visit Eruk on <planet>.`

Reviewer 3: ```suggestion
			`	"Let me tell Syciil," Rek says, then turns to her and repeats your answer. When she's done, Syciil holds her hands up with her palms toward you and moves them in a circular motion while speaking. "That's a gesture of gratitude. She says, 'Thank you. Your help will benefit both the Wanderers and Kor Efreti.' We would go with you, but both of us are busy with another project right now. If you make any progress, come back to Setar Fort and tell us." Rek and Scyiil depart. It sounds like you should visit Eruk on <planet>.`
```

				accept

> mission "Translator Development 1B"
> 	landing
> 	name "Translator Development"
> 	description "Go to <destination> to ask Eruk if his translation device could be modified for the Wanderers and Korath to use."
> 	minor
> 	source "Setar Fort"
> 	destination "Greenwater"
> 	to offer
> 		has "Wanderers: Sestor: Final: done"
> 		has "Translator Development 1A: declined"
> 	on offer
> 		conversation
> 			`As you leave your ship, you remember the conversation you had with Rek and Syciil. They asked you to go speak to Eruk on <destination> about using his translator for the Wanderers and Kor Efreti, but you were too busy to help them. Would you like to tell Rek you can help now?`
> 			choice
> 				`	(Yes.)`
> 				`	(No.)`
> 					defer
> 			`	You send a message to Rek to let her know you'll go ask Eruk about his translator. She thanks you and says to return to Setar Fort if you make any progress. Time to visit Eruk.`
> 				accept
> 		

Reviewer 3: Same issue as before.
```suggestion
```

mission "Translator Development 2A"
	landing
	name "Information Exchange"
	description "Acquire a copy of the Library of Parliament on <stopovers> for Eruk. Then, return to <destination> in exchange for information about the inner workings of his translation device."
	source "Greenwater"
	stopover "Earth"
	to offer
		or
			has "Translator Development 1A: done"
			has "Translator Development 1B: done"
	on offer
		conversation

> 			`You make contact with Eruk soon after landing, and he invites you on a tour of his facility. Compared to the last time you were here, it has doubled - maybe even tripled - in size. "The translator was released to the public a few months ago, and its reception has been fantastic," he explains. "Consequentially, my operation here has grown immensely." As the tour continues, he's sure to point out many exact details of his work. Eruk is obviously pleased with the success of his device. "But, enough about me. I suspect you're a busy person, so you likely came here with a purpose. What can I do for you?"`

Reviewer 3: Facility? Also the whole thing about releasing the translator to the public seems a bit iffy, especially when looking at Sayari's comment about the translation machines:
`She says, "There are scientists on the Hai world of <planet> who develop speaking machines, to allow the Hai who do not know the human language to communicate with humans. I will write a letter asking them to give you one of these speaking machines, so that you may talk with the Wanderers without needing me as an intermediary."`
From this, it sounds like the machine is already public and used, as well as Eruk not being the sole creator of translators, and is just part of a team.

Author: Rewritten.

			`	"I am told your Republic's governing body holds a database on <stopovers> similar to that of Alexandria. I believe it is known as the Library of Parliament. I would like to study this database as well, but I have yet to obtain a copy. I will give you the translator information in exchange for one."`
			choice
				`	"That sounds fair to me. I'll be back when I have a copy."`
					accept
				`	"Sorry, but I'm not interested in running another errand to human space for you right now. If I change my mind, I'll come back."`

> 					decline

Reviewer 2: still feel it'd just be bloating the files with the extra defer mission. if you really want the player to be able of defering the mission maybe add a branch that just jumps to a choice for "Yes(accept)" "No(defer)" if you had the mission offered before? dunno if that works though

> 					decline

Author: I'll try to play around with it and see if I can make something that'll work.

> 	on stopover

Author: Done.

	name "Data Chip Delivery"
	description "Take the translator information to Wanderer technologists on <stopovers>, then return to <destination>."
	source "Setar Fort"
	stopover "Spera Anatrusk"
	to offer
		has "Translator Development 3: done"
	on offer
		conversation
			`When you exit your ship, the spaceport is different. Every Kor Efret spaceport you've visited so far was dotted with ruined buildings and dirty alleyways long-since abandoned. One such area, however, is fenced off, and you can hear sounds of construction from behind it. Rek and Syciil emerge from a nearby building - also under construction - and approach you.`
			`	"Were you able to make any progress, Captain <last>?" Rek asks. As you explain Eruk's response and show them the data card he gave you, Rek translates to Syciil. The two have a brief conversation in Korath language, then Rek turns to you and says, "While it's unfortunate Eruk can't help us, this information is much better than nothing at all. Would you deliver the data card to Wanderer technologists on <stopovers>? I'll contact them ahead of time so they can get started right away."`
			choice
				`	"Sure, I'll go drop it off."`
					goto noquestions

> 				`	"I can do that, but before I leave, can you tell me what's going on here at the spaceport? Are they building something?"`

Reviewer 3: ```suggestion
				`	"I can do that, but can you tell me what's going on here?"`
```

> 			`	Rek smiles. "I could explain, but that would spoil the surprise. The construction will be finished by the time you return. When you're back, I'll be happy to show you around. Until then, Captain <last>." Rek and Syciil turn and head back to the building they came from. It looks like you won't get any answers until you've dropped off the data card.`

Reviewer 3: ```suggestion
			`	Rek smiles. "I could explain, but that would spoil the surprise. The construction will be finished by the time you return. When you're back, I'll be happy to show you around. Until then, <first>." Rek and Syciil turn and head back to the building they came from. It looks like you won't get any answers until you've dropped off the data card.`
```

> 			`	Rek smiles. "I could explain, but that would spoil the surprise. The construction will be finished by the time you return. When you're back, I'll be happy to show you around. Until then, Captain <last>." Rek and Syciil turn and head back to the building they came from. It looks like you won't get any answers until you've dropped off the data card.`

Author: I dunno; good point. Removed.

				accept
			label noquestions

> 			`	"Wonderful. Thank you for all your help, Captain," Rek says. She speaks to Syciil, and the two begin walking away.`

Reviewer 3: ```suggestion
			`	"Wonderful. Thank you for all your help," Rek says. She speaks to Syciil, and the two begin walking away.`
```

> 			`	Suddenly, Rek pauses in her step and turns back to you. "Captain <last>! I nearly forgot to tell you. When you're finished dropping off the information, please return to Setar Fort. I will have something you may be interested in." You nod in acknowledgement, and she turns around to catch up to Syciil.`

Reviewer 3: ```suggestion
			`	Suddenly, Rek pauses in her step and turns back to you. "<first>! I nearly forgot to tell you. When you've finished dropping off the information, please return to Setar Fort. I'll have something you may be interested in." You nod in acknowledgement, and she turns around to catch up to Syciil.`
```

				accept
	on stopover
		dialog `Sure enough, a team of Wanderer technologists is waiting for you when you arrive. You hand off the data card, and they rush away, obviously eager to dive into its contents. Time to return to <planet> and see what Rek has in store for you.`
	on complete
		payment 200000
		event "wanderers taking jobs from kor efreti"

> 		log "Brought a copy of the information used to develop my Hai translator to Wanderer researchers trying to make one for them and the Korath. The Kor Efreti have created terminals that will allow me to run jobs for them without knowing their language."

Reviewer 3: Why not put this in the on stopover?

Reviewer 3: ah, makes sense

		conversation

> 			`When you land on <planet>, the fencing is gone, and it looks like the construction is finished. Rek and Syciil meet you as you disembark. Syciil gestures with her palms toward you, moving in a circular motion again. You remember Rek told you this is a symbol of gratitude. When she is finished, she hands you <payment> and speaks in her own language. Rek translates: "She says, 'Rek and I thank you for your help in this project. This payment represents a joint contribution between the Kor Efret and Wanderer governments.'"`

Reviewer 3: ```suggestion
			`When you land on <planet>, the fencing is gone, and it looks like the construction is finished. Rek and Syciil meet you as you disembark. Syciil gestures with her palms toward you, moving in a circular motion again. When she is finished, she hands you <payment> and speaks in her own language. Rek translates: "She says, 'Rek and I thank you for your help in this project. This payment represents a joint contribution between the Kor Efret and Wanderer governments.'"`
```

			choice
				`	"I'm happy to help in whatever way I can."`
				`	"You're welcome. Do you know how long it will take for them to make a working translator?"`
			`	Rek translates your acknowledgement to Syciil, then continues speaking to you. "I spoke with the technologists after you dropped off the information. They say it will take some time to convert the Hai technology and methodology into something we would be able to use. It would then take even longer to program two new languages into the translator. From what I understand, it's too early to tell if the translator's programming would even work with Wanderer or Korath languages. I'll have to start working with them to create a database of the Korath language soon.`
			`	"All things considered, the device probably won't be done for a long time - if ever - but it's a big step in the right direction. We have you to thank for that. That brings me to my next point: I'm sure you're curious about what the new part of the spaceport is. Please, follow me." You, Rek, and Syciil walk to the newly-developed area.`

> 			`	Its centerpiece is several large, two-sided monitors. Each displays a map of local Korath space with settlements and hyperspace lanes labeled. Syciil holds her hand up to one, and a few different listings appear. Each shows the destination and displays the number of passengers and cargo. The most intuitive aspect of the setup is that it's entirely visual: there are no numbers or writing anywhere.`

Reviewer 3: > no numbers or writing

> displays the number of passengers and cargo
```suggestion
			`	The centerpiece of the area is several large, two-sided monitors. Each displays a map of local Korath space with settlements and hyperspace lanes labeled. Syciil holds her hand up to one, and a few different listings appear. Each shows the destination and displays the number of passengers and cargo. The most intuitive aspect of the setup is that it's entirely visual: there are no numbers or writing anywhere.`
```

			`	Syciil speaks through Rek. "She says, 'A translation device is not the only thing Rek and I have been developing. We've been brainstorming a way for our populations to communicate without translators should the project fall through, and this is what we've come up with. The Wanderers were kind enough to lend us the resources to build these - though it took some deliberation to determine it would not be an insult to Korath honor to accept the help. Similar terminals were recently installed in other Kor Efret systems and are being built on the new Wanderer colonies as we speak. You are welcome to use these terminals as well, Captain <last>.'" She holds her hands up, palms forward, says a few words, then departs. "She has other matters to attend to," Rek explains. "In fact, I do as well. Thanks again for all your help, <first>. The Wanderers and Kor Efreti will both benefit from it, and I hope these new jobs will work to your advantage as well. Until next time," Rek says, then she leaves to catch up with Syciil. You're left to explore the new jobs available in Korath space.`
			

> event "wanderers taking jobs from kor efreti"

Author: A small fleet has been added. I'll keep this conversation open in case others want to comment on whether they think it's necessary/too Act 3-impacting.

	news "wanderer at kor efret spaceport"
		location
			government "Kor Efret"
			not attributes "station"
			
mission "Setar Fort Heating"
	name "Supplies to <planet>"
	description "Take <cargo> from the Wanderers to the Kor Efreti on <destination>. Payment is 5,000 credits."
	minor
	cargo "heating supplies" 30
	source

> 		government "Wanderer"

Author: Done.

			`	Soon, all the supplies are off of your ship. The Korath thank you, then they begin unpacking it right in the middle of the spaceport. Already, a crowd has begun to gather around the equipment. It looks like it will be put to good use here. When you return to your ship, you see the Wanderer who offered you the job has transferred you your payment of <payment>.`
				
mission "Korath Family to Ringworld"				
	name "Korath to <planet>"
	description "A Korath family of <bunks> has paid you <payment> to take them to the Quarg ringworld of <destination>."
	minor
	passengers 3
	source "Laki Nemparu"
	destination "Kuwaru Efreti"
	to offer
		random < 60
	on offer
		conversation
			`The smog is so thick on Laki Nemparu that even the brightest hours of the day feel like evening to you. Most of the Korath are a few shades darker than those of other planets, more a result of dirtiness rather than skin pigmentation. Within minutes, a small layer of soot clings to your skin as well.`

> 			`	As you observe your surroundings, a Kor Efret approaches you followed by two others, one likely a child, based on its size, and the other hunched over a cane it uses for support - probably elderly. The one in front speaks to you rapidly in its own language before realizing you don't understand it at all. It pauses to take a deep breath, then erupts into a coughing fit as it inhales far too much smog. The older Korath behind begins coughing too, and it's almost a full minute before they've both recovered.`

Reviewer 2: why would that Kor ever assume I'd understand its language? also, earlier you used `her` and `she` for the Korath that's working with Rek, so I take it the Korath aren't like the Quarg (in that the Quarg are referred to as `it`), so maybe use `they` for this Kor or give them a gender.

			accept
			label heartless
			`	You give the credits back to the Korath, and its shoulders slump. The trio wanders off, all three coughing, until the smog envelops them. You can't help but feel a little bad.`
				decline
	on accept
		payment 500
	on complete
		conversation
			`You enjoy the new company on your ship. The child has taught you a game where you take turns throwing a ball against the wall. You spend an hour or two playing with it in the cargo hold.`
			`	When you arrive, you decide to accompany the family into the spaceport. Waiting near the docking bay are two more Korath, one an adult and the other elderly, and the three you've transported rush to meet them. After the happy reunion has died down, the Korath who had originally asked you for transportation turns to a nearby Quarg and chatters off something. The Quarg responds and then turns to you.`

> 			`	"This prideful fellow has said he regards my translation skills with reverence and asks I use them. I am inclined to acquiesce his request." This Quarg does not seem to have an especially strong grasp of your language, but you should be able to roughly gather what is being said. "The comrade says, 'I disproportionately thank you for reuniting my familial, in fateful service to our comraderie.' He additionally supplants there was no method in which this gaggle would vicinitied in which absenced your presence. That is all."`

Reviewer 2: to me it seems the first sentence isn't nearly as tough to understand as the rest of what the Quarg says; since it's after that one and another sentence that the game tells you that `This Quarg does not seeem to have an especially strong grasp of your language`, maybe make the first one a bit more convoluted like the rest
//
when googling `comraderie` I only get that word from Urban Dictionary, so pointing it out in case it's not really a word but a typo of camaraderie or something else.

			choice

> 				`	"Tell him it's no problem at all."`

Reviewer 2: yah since the Korath does have a gender then that instance mentioned before should be changed accordingly to these choices

				`	"Tell him he has a wonderful family, and I was glad to help."`

> 			`	The Quarg nods and turns to translate. The Korath responds while holding its palms toward you and moving them in a wide circular motion. "This is indicating of thankfulness of wholesomest measure. He thanks you for your fortunate words and wishes you kind travels." You acknowledge the response, thank the Quarg for its (rough) translation services, and return to your ship."`

Reviewer 2: `its palms` `He thanks you` conflicting stuffs again

			`	The one in front turns back to where it came from, and the Korath behind you hisses and gestures forward. It looks like you should follow. You brace yourself and turn the corner to see - animals?`
			`	The rusting buildings have given way to a massive abandoned plaza, and the entire space is taken up by a mix of large livestock and supplies. They remind you of sheep, but they're around five times larger, have six legs, and come in a variety of brown-black colors. Each has a full coat of fur and an additional blanket made of their own wool-like fabric on them. It's the same material the Korath - the one who led you here, the three behind you, and three more walking around tending to the herd - are wearing. Even with their extra blanket, the "sheep" are huddled in small groups and look uncomfortably cold. They occasionally bleat loudly whenever a cold wind rushes through the plaza. Fortunately for you, these Kor Efreti look to be farmers, not muggers. This must have been the only place near the spaceport that they could keep their entire herd together.`

> 			`	The one who led you here, who also seems to be the leader, gestures for you to come over to it. It's opened a paper map - a rare sight in the era of interstellar travel. When you take a look, you realize it's not of the local area, but space: it seems to show the surroundings systems. The Korath points to one, pauses and gestures broadly at the mix of livestock and supplies, then draws its finger to another system and says something that sounds roughly like Karek Fornati. "Karek Fornati?" you confirm, and the Korath hisses in approval. It looks like they want transportation there. You wager the livestock and supplies will take up roughly 40 tons, and there are seven Korath.`

Reviewer 2: could use `<planet>`
